manifests/addons/metrics-emitter.yml

```diff
@@ -0,0 +1,44 @@
+instance_groups:
+- name:      rmq-broker
+  jobs:
+  - name: loggregator_agent
+    release: loggregator-agent
+    consumes:
+      doppler:
+        from: doppler
+        deployment: (( grab meta.cf.deployment_name ))
+    properties:
+      loggregator:
+        tls:
+          ca_cert: (( vault meta.cf.exodus_path ":loggregator_ca" ))
+          agent:
+            cert: (( vault meta.cf.exodus_path ":loggregator_tls_agent_cert" ))
+            key: (( vault meta.cf.exodus_path ":loggregator_tls_agent_key" ))
+  - name: rabbitmq-metrics-emitter
+    release: rabbitmq-metrics-emitter
+    properties:
+      rabbitmq_metrics_emitter:
+        cloud_foundry:
+          api:    (( concat "https://api." meta.cf.system_domain ))
+          skip_ssl_validation: true
+          username:   (( vault meta.cf.exodus_path ":admin_username" ))
+          password:   (( vault meta.cf.exodus_path ":admin_password" ))
+        rmq_management:
+          endpoint: (( concat "http://" params.rmq_domain ":15672/api" ))
+          user: management
+          password: (( vault meta.vault "/rabbitmq/admin/management:password" ))
+        loggregator:
+          tls:
+            cert: (( vault meta.cf.exodus_path ":loggregator_tls_agent_cert" ))
+            key: (( vault meta.cf.exodus_path ":loggregator_tls_agent_key" ))
+            ca_cert: (( vault meta.cf.exodus_path ":loggregator_ca" ))
```

Comment:
out of curiosity, why is there a loggregator agent on the box, as well as creds for the tool to talk to loggregator directly?

Reply:
Talking to loggregator happens via localhost if I remember correctly